test/container/1.6.0/Dockerfile.dlc.cpu

```diff
@@ -1,6 +1,16 @@
 ARG region
 FROM 763104351884.dkr.ecr.$region.amazonaws.com/mxnet-training:1.6.0-cpu-py2
 
+# OpenSHH config for MPI communication
```

Comment:
should we add a TODO here to note that this can be removed once the DLC images have OpenSSH installed?

Reply:
added.

Reply:
Removing it in this PR: https://github.com/aws/sagemaker-mxnet-training-toolkit/pull/182
Now that DLC has been released.